wp-cldr.php

```diff
@@ -10,6 +10,139 @@
  * Domain Path: /languages/
  * License:     GPLv2 or later
  * License URI: https://www.gnu.org/licenses/gpl-2.0.html
+ *
+ * @package    wp-cldr
  */
 
 require_once plugin_dir_path( __FILE__ ) . 'class.wp-cldr.php';
+require_once( ABSPATH . 'wp-admin/includes/translation-install.php' );
+
+add_action( 'admin_menu', 'wp_cldr_menu' );
+
+/**
+ * Gets the wp-admin menu info.
+ */
+function wp_cldr_menu() {
+	add_options_page(
+		'WP CLDR Options',
+		'WP CLDR',
+		'manage_options',
+		'wp-cldr',
+		'wp_cldr_settings'
+	);
+}
+
+/**
+ * Gets the settings and examples page for the plugin.
+ */
+function wp_cldr_settings() {
+
+	if ( isset( $_GET['locale'] ) && 1 < strlen( sanitize_text_field( wp_unslash( $_GET['locale'] ) ) ) ) {
+		$locale = sanitize_text_field( wp_unslash( $_GET['locale'] ) );
+	} else {
+		$locale = get_locale();
+	}
+	if ( isset( $_GET['country'] ) && 2 === strlen( sanitize_text_field( wp_unslash( $_GET['country'] ) ) ) ) {
+		$country = sanitize_text_field( wp_unslash( $_GET['country'] ) );
+	} else {
+		$country = 'US';
+	}
+	$cldr = new WP_CLDR( $locale );
+	$locales = $cldr->languages_by_locale();
+	$languages = get_available_languages();
+	$translations = wp_get_available_translations();
+	$territories = $cldr->territories_by_locale();
+	?>
+
+	<div class="wrap">
+	<h1><?php esc_html_e( 'WP CLDR settings and info', 'wp-cldr' ); ?></h1>
+	<?php esc_html_e( 'This plugin provides WordPress developers with easy access to localized country/region names, language names, currency names/symbols/usage, and other localization info from the Unicode Common Locale Data Repository (CLDR). Here are some examples:', 'wp-cldr' );
+	echo '<br>'; ?>
+
+	<form method="get" name='locale'>
+	<table width="60%">
+	<tr>
+		<th width="40%" align="left"><label><?php esc_html_e( 'WordPress locales', 'wp-cldr' ); ?></label></th>
+		<td>
+			<?php
+			wp_dropdown_languages( array(
+				'name'         => 'locale',
+				'id'           => 'locale',
+				'selected'     => $locale,
+				'languages'    => $languages,
+			) );
+			?>
+		</td>
+		<td>
+			<input type="hidden" name="page" value="wp-cldr">
+			<input type="hidden" name="country" value="<?php esc_attr_e( $country ); ?>">
+			<?php submit_button( 'Update examples', 'secondary' ); ?>
+		</td>
+	</tr>
+	</table>
+	</form>
+
+	<?php
+
+	esc_html_e( 'WordPress locale:', 'wp-cldr' );
+	echo ' <code> ' . esc_html( $locale ) . '</code> ' . esc_html( $translations[ $locale ]['native_name'] ) . ' / ' . esc_html( $translations[ $locale ]['english_name'] ) . '<br>';
```

Comment:
Getting an undefined index notice here when set to English / default. Screenshot:
https://cloudup.com/c1TouVZ5vnT